vital/types/image.cxx

```diff
@@ -40,6 +40,25 @@
 namespace kwiver {
 namespace vital {
 
+template <typename T> VITAL_EXPORT
+image_pixel_traits::pixel_type const image_pixel_traits_of<T>::static_type;
+
+template struct image_pixel_traits_of<uint8_t>;
```

Comment:
I'm curious why these explicit instantiations are now needed.

Reply:
```
arrows/vxl/tests/CMakeFiles/test-vxl-image.dir/test_image.cxx.o: In function `testing::AssertionResult testing::internal::EqHelper<false>::Compare<kwiver::vital::image_pixel_traits::pixel_type, kwiver::vital::image_pixel_traits::pixel_type>(char const*, char const*, kwiver::vital::image_pixel_traits::pixel_type const&, kwiver::vital::image_pixel_traits::pixel_type const&)':
/usr/include/gtest/gtest.h:1485: undefined reference to `kwiver::vital::image_pixel_traits_of<signed char>::static_type'
```

...which comes from:
```
  EXPECT_EQ( image_pixel_traits_of<T>::static_type, vimg.pixel_traits().type );
```

(...and another equivalent assertion). Basically, something about the guts of `EXPECT_EQ` needs that to be an actual, exported symbol, which it wasn't before.